webapp/channels/src/components/air_gapped_contact_sales_modal/index.tsx

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useCallback, useMemo} from 'react';
+import {useIntl} from 'react-intl';
+import {useDispatch} from 'react-redux';
+
+import {GenericModal} from '@mattermost/components';
+
+import {closeModal} from 'actions/views/modals';
+
+import ExternalLink from 'components/external_link';
+
+import accessProblemImage from 'images/access_problem.svg';
+import {ModalIdentifiers} from 'utils/constants';
+
+import './style.scss';
+
+function AirGappedContactSalesModal() {
+    const dispatch = useDispatch();
+    const {formatMessage} = useIntl();
+
+    const handleOnClose = useCallback(() => {
+        dispatch(closeModal(ModalIdentifiers.AIR_GAPPED_CONTACT_SALES));
+    }, []);
+
+    const airGappedContactSalesLink = useMemo(() => {
+        return (
+            <ExternalLink
+                location='air_gapped_contact_sales_modal'
+                href='https://mattermost.com/contact-sales/'
+            >
+                {'https://mattermost.com/contact-sales/'}
+            </ExternalLink>
+        );
+    }, []);
```

Comment:
This doesn't need a use memo, since it is not passed as a prop. It could be directly put in the return.

Reply:
Done!